client/profile-wizard/header.js

```diff
@@ -71,29 +72,30 @@ export default class ProfileWizardHeader extends Component {
 			( step ) => step.key === currentStep
 		);
 
-		visibleSteps.map( ( step, index ) => {
+		visibleSteps.forEach( ( step, index ) => {
```

Comment:
Since we don't assign the return value to a variable, I think we just need to use the `forEach` here.